python/tests/transformers/tf_image_test.py

```diff
@@ -43,11 +43,12 @@ def _loadImageViaKeras(self, raw_uri):
         image = np.expand_dims(image, axis=0)
         return preprocess_input(image)
 
+    # TODO: I believe this is already tested in named_image_test, should we remove it?
     def test_load_image_vs_keras(self):
         g = tf.Graph()
         with g.as_default():
             image_arr = utils.imageInputPlaceholder()
```

Comment:
is it an invariant that imageInputPlaceholder() is a BGR image, or we use it as a placeholder for any 3-channel image?

Reply:
I don't think that is specified here so it's any image (you can set number of channels too). 
I think this is too low level interface, order should be specified by the caller.